common/vendor/MarkdownEditor/angular-markdown-editor.js

```diff
@@ -0,0 +1,200 @@
+angular

```

Comment:
Description for the directive has been added. Commit 40c91da906c89793b6dfb30d25781a1b21523bc1